config/settings.yml

```diff
@@ -579,6 +615,7 @@ development: &development
       - state
       - lifecycle
       - credentials
+      - multi_tenancy
```

Comment:
In service broker agent we need to add extra APIs to support lifecycle operations of virtual hosts. Collectively these will the APIs to support "multi_tenancy" feature. That's why it is required to add a new feature.

Reply:
Since VHOSTManager is specific to RabbitMq, is there any RabbitMQ Dedicated service plan which does not support multi_tenancy? Or is it supposed to be supported by all plans? If all plans of RabbitMQ must support it, then should this be required ?

Reply:
Currently all the RabbitMQ 3.6 plans will support multi-tenancy. 
With RabbitMQ 3.7 we expect a better support for multi tenancy. 
Although currently we do not provide any plan with RabbitMQ 3.7, in future we will do, and then there is a possibility that some of the existing plans can withdraw support for multi-tenancy. Not fully sure though.